newsletter/templates/newsletter/message/message.txt

```diff
@@ -12,4 +12,4 @@
 
 ++++++++++++++++++++
 
-Unsubscribe: http://{{ site }}{% url "newsletter_unsubscribe_request" newsletter.slug %}
+{% trans "Unsubscribe:" %} http://{{ site }}{% url "newsletter_unsubscribe_request" newsletter.slug %}

```

Comment:
But in this case we should also include word order and include the URL in the message as a substitution. I'm not sure if we want to go that detailed, I would like to leave the decision up to you.